modules/contrib/site_verify/site_verify.admin.inc

```diff
@@ -180,11 +185,49 @@ function site_verify_edit_form($form, &$form_state, $record = array(), $engine =
 }
 
 /**
- * Validation callback; save the uploaded file and check file name uniqueness.
+ * Validation callback; sanitize metatag and save the uploaded file
+ */
+function site_verify_validate($form, &$form_state) {
+  if (isset($form_state['values']['meta']) && $form_state['values']['meta']) {
+    site_verify_sanitize_metatag($form, &$form_state);
```

Comment:
Changed this to use a standard form validation function vs. a custom like the module was doing with http://cgit.drupalcode.org/site_verify/tree/site_verify.admin.inc?h=7.x-1.x#n160